nanoDFT/experimental_pmap_nanoDFT.py

```diff
@@ -215,28 +218,56 @@ def nanoDFT(mol, opts):
     # we can have "ipus" argument instead, or pod16 as backend? 
     if opts.backend == "ipu":
         jitted_nanoDFT = jax.pmap(partial(_nanoDFT, opts=opts, mol=mol), axis_name="p", backend=opts.backend,
-        in_axes=(None, None, None, None, None, None, None, None, None, None, None, (None, None, None), 0, 0, [0, 0, 0]))
-
-        def sparse_representation(ERI):
-            rows, cols = np.nonzero(ERI)
-            values     = ERI[rows, cols]
-            return rows, cols, values 
-
-        # ERI = tensors[6]  # ERI is just the 6th element in tensors
-        sparse_ERI = sparse_representation(ERI)
-        remainder = sparse_ERI[-1].shape[0] % 16
-        if remainder != 0:
-            sparse_ERI = [np.pad(a, (0, -remainder+16)) for a in sparse_ERI]
-        sparse_ERI = [a.reshape(16, -1) for a in sparse_ERI]
-        # tensors[6] = sparse_ERI
+        in_axes=(None, None, None, None, None, None, None, None, None, None, None, (None, None, None), 0, 0, [0, 0]))
+
+        if False: 
```

Comment:
Unreachable block, should we remove it or have a flag to control it?

Reply:
Removing.